be/src/exec/olap_scan_node.cpp

```diff
@@ -1503,6 +1503,7 @@ void OlapScanNode::transfer_thread(RuntimeState* state) {
      * 4. Regularly increase the priority of the remaining tasks in the queue to avoid starvation for large queries
      *********************************/
     PriorityThreadPool* thread_pool = state->exec_env()->scan_thread_pool();
+    PriorityThreadPool* s3_thread_pool = state->exec_env()->s3_scan_thread_pool();
```

Comment:
Use remote_thread_pool, it is not only for S3

Reply:
done